packages/opentelemetry-exporter-prometheus/src/prometheus.ts

```diff
@@ -29,7 +29,10 @@ import { Counter, Gauge, labelValues, Metric, Registry } from 'prom-client';
 import * as url from 'url';
 import { ExporterConfig } from './export/types';
 import { LabelSet } from '@opentelemetry/metrics/build/src/LabelSet';
-import { CounterSumAggregator } from '@opentelemetry/metrics/build/src/export/Aggregator';
+import {
+  CounterSumAggregator,
+  ObserverAggregator,
+} from '@opentelemetry/metrics/build/src/export/Aggregator';
```

Comment:
Import from the static path is not right, I have opened separate PR (#831) to fix this.